core/modules/comment/comment.theme.inc

```diff
@@ -106,9 +106,16 @@ function template_preprocess_comment(&$variables) {
   $uri['options'] += array('attributes' => array('class' => array('permalink'), 'rel' => 'bookmark'));
 
   $variables['title'] = l($comment->subject, $uri['path'], $uri['options']);
+  // In case title is hidden, we can use 'created' as permalink
+  $variables['created_permalinked'] = l($variables['created'], $uri['path'], $uri['options']);
```

Comment:
What about instead of adding a new parmalink version of the submitted date, we make `$uri['path']` a variable that themes can use to create permalinks of whatever they want? E.g.:

```php
$variables['permalink_path'] = $uri['path']);
```

Reply:
I added this to discuss as a todo in original issue.